packages/e2e-tests/fixtures/blocks/core__pullquote__main-color.json

```diff
@@ -1,16 +1,21 @@
 [
-	{
-		"clientId": "_clientId_0",
-		"name": "core/pullquote",
-		"isValid": true,
-		"attributes": {
-			"value": "<p>Pullquote custom color</p>",
-			"citation": "my citation",
-			"customMainColor": "#2207d0",
-			"textColor": "subtle-background",
-			"className": "has-background is-style-default"
-		},
-		"innerBlocks": [],
-		"originalContent": "<figure class=\"wp-block-pullquote has-background is-style-default\" style=\"border-color:#2207d0\"><blockquote class=\"has-text-color has-subtle-background-color\"><p>Pullquote custom color</p><cite>my citation</cite></blockquote></figure>"
-	}
+    {
+        "clientId": "_clientId_0",
+        "name": "core/pullquote",
+        "isValid": true,
+        "attributes": {
+            "value": "<p>Pullquote custom color</p>",
+            "citation": "before block supports",
```

Comment:
The citation text is a bit misleading here, no? Since this isn't really a deprecation fixture, but a "regular" one.

Also, should we maybe rename this set of tests to `core__pullquote__color.*` (or `text-color`, `background-color`,...?) -- since the attribute isn't called `mainColor` any more?

Reply:
That was a copy 🍝 oversight. 👍

Reply:
> Also, should we maybe rename this set of tests to core__pullquote__color.* (or text-color, background-color,...?) -- since the attribute isn't called mainColor any more?

Good idea, thanks. 

I went with `*__custom-colors__* to test the new style attribute against the deprecation of `mainColor` & `customMainColor`, which (I think) is more in the spirit of the [original fixture](https://github.com/WordPress/gutenberg/blob/trunk/packages/e2e-tests/fixtures/blocks/core__pullquote__main-color.html).